tests/tests_shared.c

```diff
@@ -261,7 +261,7 @@ size_t sir_readline(FILE* f, char* buf, size_t size) {
 
     while (idx < size) {
         int ch = getc(f);
-        if (EOF == ch || '\n' == ch)
+        if (EOF == ch || '\x0a' == ch || '\x0d' == ch)
```

Comment:
Y

Reply:
Just did a universal search and replace across all files for \r and \n and replaced with proper macro and/or \x0a or \x0d, repeating until there was no more \r or \n.  On this one particular, I check for both because either can be used to end the line depending on the terminal type (mostly old stuff like A/UX using on \r though).  I'll keep \r \n if you prefer tho.